### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by jpaadre**
Hey Nick,

Here's my pull request. I've cleaned up the spelling/typos and made it easier to read. 

I haven't added any new SCTs (as outlined in the Evernote doc). I could give those a shot if you'd like.

Jeff

### CODE REVIEW

File: chapter3.Rmd [hunk 1]
```diff
@@ -216,7 +216,7 @@ success_msg("Great! You're on the way of becoming an R jedi! Continue to the nex
 --- type:NormalExercise xp:100
 ## Calculating the worldwide box office 
 
-The single most important thing for a movie in order to become an instant legend in Tinseltown is its worldwide box office figures. 
+The single most important thing for a movie in order to become an instant legend in Tinseltown, is its worldwide box office figures.
```
Comment (ncarchedi) [inline chapter3.Rmd]: I'd leave the comma off here. You can put it back in a new commit and the pull request will automatically update when you push to GitHub.

File: chapter3.Rmd [hunk 2]
```diff
@@ -301,7 +301,7 @@ Bind the `worldwide_vector` to the `star_wars_matrix` with the `cbind()` functio
 ```
 cbind( the_correct_matrix, the_correct_vector)
 ```
-Assign the result to `all_star_wars_matrix`.
+Assign the result to `all_wars_matrix`.
```
Comment (ncarchedi) [inline chapter3.Rmd]: Eek, good catch!

File: chapter4.Rmd [hunk 3]
```diff
@@ -361,7 +361,7 @@ success_msg("Phew, it seems that R is gender-neutral. Or maybe it just wants to
 --- type:NormalExercise xp:100
 ## Ordered factors
 
-Since `"Male"` and `"Female"` are unordered (or nominal) factor levels, R returns a warning message, telling you that the greater than operator is not meaningful. As seen before, R attaches an equal value to the levels for such factors. 
+Since `"Male"` and `"Female"` are non-ordered (or nominal) factor levels, R returns a warning message, telling you that the greater than operator is not meaningful. As seen before, R attaches an equal value to the levels for such factors.
```
Comment (ncarchedi) [inline chapter4.Rmd]: Keep the word "unordered" here. It's actually a technical term for a factor variable in R where order doesn't matter. For example, Male and Female are unordered, but Unsatisfied/Neutral/Satisfied would be ordered factors, etc.

File: chapter4.Rmd [hunk 4]
```diff
@@ -412,7 +412,7 @@ success_msg("A job well done! Continue to the next exercise.")
 --- type:NormalExercise xp:100
 ## Ordered factors (2)
 
-`speed_vector` should be converted to an ordinal factor since its categories have a natural ordening. By default, the function [`factor()`](http://www.rdocumentation.org/packages/base/functions/factor) transforms `speed_vector` into an unordered factor. To create an ordered factor, you have to add two additional arguments: `ordered` and `levels`. 
+`speed_vector` should be converted to an ordinal factor since its categories have a natural ordering. By default, the function [`factor()`](http://www.rdocumentation.org/packages/base/functions/factor) transforms `speed_vector` into an non-ordered factor. To create an ordered factor, you have to add two additional arguments: `ordered` and `levels`.
```
Comment (ncarchedi) [inline chapter4.Rmd]: Keep "unordered" here as well :)

File: chapter4.Rmd [hunk 5]
```diff
@@ -481,7 +481,7 @@ success_msg("Great! Have a look at the console. It is now indicated that the Lev
 
 Having a bad day at work, 'data analyst number two' enters your office and starts complaining that 'data analyst number five' is slowing down the entire project. Since you know that 'data analyst number two' has the reputation of being a smarty-pants, you first decide to check if his statement is true. 
 
-The fact that `factor_speed_vector` is now ordered enables us to compare different elements (the data analysts in this case). You can simply do this by using the well-known operators.
+The fact that `factor_speed_vector` is now ordered, enables us to compare different elements (the data analysts in this case). You can simply do this by using the well-known operators.
```
Comment (ncarchedi) [inline chapter4.Rmd]: I'd actually leave the comma off here

File: chapter5.Rmd [hunk 6]
```diff
@@ -58,7 +58,7 @@ Wow, that is a lot of cars!
 Working with large data sets is not uncommon in data analysis. When you work with (extremely) large data sets and data frames, your first task as a data analyst is to develop a clear understanding of its structure and main elements. Therefore, it is often useful to show only a small part of the entire data set. 
 
 So how to do this in R?
-Well, the function [`head()`](http://www.rdocumentation.org/packages/utils/functions/head) enables you to show the first observations of a data frame (or any R object you pass to it). Unoriginally, the function [`tail()`](http://www.rdocumentation.org/packages/utils/functions/head) prints out the last observations in your data set.
+Well, the function [`head()`](http://www.rdocumentation.org/packages/utils/functions/head) enables you to show the first observations of a data frame (or any R object you pass to it). Un-originally, the function [`tail()`](http://www.rdocumentation.org/packages/utils/functions/head) prints out the last observations in your data set.
```
Comment (ncarchedi) [inline chapter5.Rmd]: I think you can leave it as "unoriginal", see https://www.dropbox.com/s/t2n7d77oevhqo3s/Screenshot%202015-12-23%2011.52.15.png?dl=0

**Comment by ncarchedi** [comment]
@filipsch Ready to merge?

**Comment by filipsch** [comment]
Looks good!

**Pull request merged**